upup/pkg/fi/cloud.go

```diff
@@ -39,6 +39,12 @@ type Cloud interface {
 
 	// GetCloudGroups returns a map of cloud instances that back a kops cluster
 	GetCloudGroups(cluster *kops.Cluster, instancegroups []*kops.InstanceGroup, warnUnmatched bool, nodes []v1.Node) (map[string]*cloudinstances.CloudInstanceGroup, error)
+
+	// DetachInstance detaches instance from ASG (aws implememtation only)
```

Comment:
remove text ` (aws implememtation only)` as I see it in future we could implement something similar to other providers. And dummy interfaces are already added